### PR TITLE
Features

### PR DESCRIPTION
**Pull request by 0bafemi**
Created The Full Database for UniBiz

### CODE REVIEW

File: src/components/login-signup/login-signup.jsx [hunk 1]
```diff
@@ -2,37 +2,46 @@ import React, { useState } from 'react';
...
+      const body = isSignUp ? JSON.stringify({ email, password, businessName }) : JSON.stringify({ email, password });
+      if (email === '') {
+        setError('Please enter an email address');
+        return;
+    }
+      // Check if the email address ends with ".edu"
+      if (!email.endsWith('.edu')) {
+          setError('This is not a valid college email address');
+          return;
+      }
```
Comment (winstongrocholski) [inline src/components/login-signup/login-signup.jsx]: Do all collage emails have .edu in them?

File: src/components/login-signup/login-signup.jsx [hunk 2]
```diff
@@ -2,37 +2,46 @@ import React, { useState } from 'react';
...
+              method: 'POST',
+              headers: {
+                  'Content-Type': 'application/json'
+              },
+              body: body
+          });
+          const data = await response.json();
+          if (response.ok) {
+              localStorage.setItem('token', data.token);
+              // Redirect or update state to indicate success
```
Comment (winstongrocholski) [inline src/components/login-signup/login-signup.jsx]: This feels like a weird place for a comment. I can not tell if this comment is for the `else` block below it or if it should be for the code above.

File: src/components/login-signup/login-signup.jsx (hunk 1 above)
Comment (0bafemi) [reply src/components/login-signup/login-signup.jsx]: Yes all colleges to my knowledge have the .edu suffix at the end of them. checking using this was a suggestion by Daniel, as opposed to using a school email verification API

File: src/components/login-signup/login-signup.jsx (hunk 2 above)
Comment (0bafemi) [reply src/components/login-signup/login-signup.jsx]: I apologize for the confusion the comment was meant to indicate that after a successful sign up my user should be redirected to the log in page. I would work on better comment placements in subsequent pr's

**Pull request merged**